src/structures/InviteGuild.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const Guild = require('./Guild');
+const WelcomeScreen = require('./WelcomeScreen');
+
+/**
+ * Represents a guild received from an invite, including welcome screen data if available.
+ * @extends {Guild}
+ */
+class InviteGuild extends Guild {
```

Comment:
Maybe this could extend `BaseGuild` now, since #5472 was merged?
That way we wouldn't document unavailable properties anymore.